test/e2e_node/remote/node_e2e.go

```diff
@@ -101,23 +102,54 @@ func prependMemcgNotificationFlag(args string) string {
 	return "--kubelet-flags=--kernel-memcg-notification=true " + args
 }
 
-// updateOSSpecificKubeletFlags updates the Kubelet args with OS specific
-// settings.
-func updateOSSpecificKubeletFlags(args, host, workspace string) (string, error) {
-	output, err := SSH(host, "cat", "/etc/os-release")
+// osSpecificActions takes OS specific actions required for the node tests
+func osSpecificActions(args, host, workspace string) (string, error) {
+	output, err := getOSDistribution(host)
 	if err != nil {
 		return "", fmt.Errorf("issue detecting node's OS via node's /etc/os-release. Err: %v, Output:\n%s", err, output)
 	}
 	switch {
-	case strings.Contains(output, "ID=gci"), strings.Contains(output, "ID=cos"):
+	case strings.Contains(output, "gci"), strings.Contains(output, "cos"):
 		args = prependMemcgNotificationFlag(args)
 		return prependCOSMounterFlag(args, host, workspace)
-	case strings.Contains(output, "ID=ubuntu"):
+	case strings.Contains(output, "ubuntu"):
 		return prependMemcgNotificationFlag(args), nil
+	case strings.Contains(output, "fedora"):
```

Comment:
Might as well get rhel and centos as well.

Reply:
Thanks, I will add those too.

Reply:
I added the check for, 

1. Fedora CoreOS and Fedora  - `fedora`
2. CentOS - `centos`
3. RHEL - `rhel` 
4. Red Hat Core OS - `rhcos`